science/projects/zahrasadat_sajjadi/01-main-content/zarebin_p1.ipynb

```diff
@@ -0,0 +1,2717 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #81.        if training_mode == True:
</pre><p>Two consecutive for loops can be written in the form of <strong>a for loop</strong> with two <strong>AND conditions</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>